### PR TITLE
Update technologies.md

### PR DESCRIPTION
**Pull request by dannoakes**
Adding the Windows technologies used.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/intro/technologies.md [hunk 1]
```diff
@@ -32,12 +32,31 @@ Win32 Applications |	Teams is not shipped with the device, but is packaged and p
...
 
 
-## Windows 10 Enterprise E3 or E5
+## Windows 10 Enterprise E3, E5 and Microsoft Defender for Endpoint
+ --- | --- 
+Recommended
+[Windows Hello for Business](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-identity-verification) | Customers are recommended to implement Windows Hello for Business to replace passwords with strong two-factor authentication used on Microsoft Managed Desktop devices.
+[Application Virtualization](https://docs.microsoft.com/windows/application-management/app-v/appv-technical-reference) | Customers can deploy Application Virtualization (App-V) packages using the Intune Win32 app management client.
+[Microsoft 365 data loss prevention](https://docs.microsoft.com/microsoft-365/compliance/endpoint-dlp-learn-about) | Customers are recommend to implement Microsoft 365 data loss prevention (DLP) to monitor the actions that are being taken on items you've determined to be sensitive and to help prevent the unintentional sharing of those items.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/technologies.md:42]:
```suggestion
[Microsoft 365 data loss prevention](https://docs.microsoft.com/microsoft-365/compliance/endpoint-dlp-learn-about) | Customers are recommended to implement Microsoft 365 data loss prevention (DLP) to monitor the actions that are being taken on items you've determined to be sensitive and to help prevent the unintentional sharing of those items.   
```

File: microsoft-365/managed-desktop/intro/technologies.md [hunk 2]
```diff
@@ -32,12 +32,31 @@ Win32 Applications |	Teams is not shipped with the device, but is packaged and p
...
+[Application Virtualization](https://docs.microsoft.com/windows/application-management/app-v/appv-technical-reference) | Customers can deploy Application Virtualization (App-V) packages using the Intune Win32 app management client.
+[Microsoft 365 data loss prevention](https://docs.microsoft.com/microsoft-365/compliance/endpoint-dlp-learn-about) | Customers are recommend to implement Microsoft 365 data loss prevention (DLP) to monitor the actions that are being taken on items you've determined to be sensitive and to help prevent the unintentional sharing of those items.   
+
+Included in the service plan
+[BitLocker Drive Encryption](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview) | BitLocker Drive Encryption is used to encrypt all system drives. 
+[Windows Defender System Guard]( https://docs.microsoft.com/windows/security/threat-protection/windows-defender-system-guard/system-guard-how-hardware-based-root-of-trust-helps-protect-windows) | Protects the integrity of the system at start up and validates that system integrity has truly been maintained.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/technologies.md:48]:
```suggestion
[Windows Defender System Guard]( https://docs.microsoft.com/windows/security/threat-protection/windows-defender-system-guard/system-guard-how-hardware-based-root-of-trust-helps-protect-windows) | Protects the integrity of the system at start-up and validates that system integrity has truly been maintained.
```

File: microsoft-365/managed-desktop/intro/technologies.md [hunk 3]
```diff
@@ -32,12 +32,31 @@ Win32 Applications |	Teams is not shipped with the device, but is packaged and p
...
+[Microsoft Defender for Endpoint | Network Protection](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/network-protection) | Network protection expands the scope of Microsoft Defender SmartScreen to block all outbound HTTP(s) traffic that attempts to connect to low-reputation sources.
+[Microsoft Defender Tamper Protection](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-antivirus/prevent-changes-to-security-settings-with-tamper-protection) | Windows Tamper Protection is used to prevent security settings such as anti-virus protection from being changed.
+[Microsoft Defender Antivirus Behavior-based, heuristic, and real-time antivirus protection]( https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-antivirus/microsoft-defender-antivirus-in-windows-10) | Always on scanning for file and process threats which may not be detected as malware.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/technologies.md:57]:
```suggestion
[Microsoft Defender Antivirus Behavior-based, heuristic, and real-time antivirus protection]( https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-antivirus/microsoft-defender-antivirus-in-windows-10) | Always-on scanning for file and process threats which may not be detected as malware.
```

File: microsoft-365/managed-desktop/intro/technologies.md [hunk 4]
```diff
@@ -32,12 +32,31 @@ Win32 Applications |	Teams is not shipped with the device, but is packaged and p
 Web Applications |	Yammer, Office in a browser, Delve, Flow, StaffHub, PowerApps, and Planner are not shipped with the device. Users can access the web version of these applications with a browser.
 
 
-## Windows 10 Enterprise E3 or E5
+## Windows 10 Enterprise E3, E5 and Microsoft Defender for Endpoint
+ --- | ---
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/technologies.md]:
```suggestion
 --- | --- |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@dannoakes

**Comment by yogkumgit** [comment]
@dannoakes Please implement the feedback so that merge can be initiated. Thanks

**Comment by dannoakes** [comment]
@yogkumgit please confirm the feedback.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@dannoakes - please check, if you look at the rich diff, the table does not render correctly.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jaimeo** [comment]
@dannoakes your revised heading is unclear. Are there three things in the list? (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5, 3) Microsoft Defender for Endpoint.) Or two: (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5 + Microsoft Defender for Endpoint)?

**Comment by yogkumgit** [comment]
@dannoakes Please reply to this comment. Thanks

**Comment by dannoakes** [comment]
> @dannoakes your revised heading is unclear. Are there three things in the list? (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5, 3) Microsoft Defender for Endpoint.) Or two: (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5 + Microsoft Defender for Endpoint)?

It is s single heading "Windows 10 Enterprise E3, E5 and Microsoft Defender for Endpoint" for the table. It lists the 3 product SKUs a customer may have purchased. If it is simpler change to "Windows 10 Enterprise E5 or E3 with Microsoft Defender for Endpoint"

This is no different to the "Office 365 E3 or E5" heading for the above table, this is describing 2 product SKUs a customer may have purshased.

**Comment by jaimeo** [comment]
Thanks! I think that slight rewording makes it clearer, especially for people who are just scanning.

**Pull request merged**